modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/resources/META-INF/resources/mfa_sample_checker/verify_browser.jsp

```diff
@@ -0,0 +1,27 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<h3>
+	<liferay-ui:message key="please-enter-something" />:
+</h3>
+
+<aui:input label="mfa-sample" name="mfaSample" showRequiredLabel="yes" />
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: